modules/phase_field/test/tests/grain_growth/tests

```diff
@@ -100,10 +100,20 @@
     requirement = 'The system shall support a faux voronoi tesselation grain structure IC without using FeatureFloodCount when
                   the number of grains equal to the number of order parameters'
     design = 'FauxPolycrystalVoronoi.md'
-    cli_args ='UserObjects/voronoi/type=PolycrystalVoronoi'
+    cli_args ='UserObjects/voronoi/type=FauxPolycrystalVoronoi'
     issues = '#14697'
   [../]
 
+  [./mesh_adaptivity]
+    type = 'Exodiff'
+    input = 'voronoi_adaptivity.i'
+    exodiff = 'voronoi_adaptivity_out.e'
+    min_parallel = 2
+    requirement = 'Distributed mesh generator should work with the mesh adaptivity when solving phase field equations'
```

Comment:
I'd recommend rewording this to say:

"The system shall be able to apply mesh adaptivity and solve phase field equations on a mesh generated in parallel."